app/views/users/index.html.erb

```diff
@@ -6,6 +6,22 @@
         <span class="profile-link">
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
         </span>
+        <% if current_user != user %>
```

Comment:
- [ ] There should be no logic in views.
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)

Let's try to clean it:
- we can get rid of this `if current_user != user`, by only sending what we need from out controller. You are trying to exclude the current_user from rendering, so instead of sending `@users = User.all` in index action of `users_controller`, we can send only all users except the current_user. Like this: `@users = User.where.not(current_user)`

Controllers should gather, organize, and filter data that should be available in views. 
Views should be dumb.

The other views logic should be encapsulated in helpers methods and used in here. Take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)